ui/app/components/clients/dashboard.js

```diff
@@ -9,9 +9,28 @@ export default class Dashboard extends Component {
     { key: 'distinct_entities', label: 'unique entities' },
     { key: 'non_entity_tokens', label: 'non-entity tokens' },
   ];
-  @tracked selectedNamespace = null;
+  // For startDate Modal
+  months = Array.from({ length: 12 }, (item, i) => {
```

Comment:
We could use the const `DATE_MAPPING` here if it was in a util file - might make sense if there are other shared functions between files